source/trackers.json

```diff
@@ -212,6 +212,12 @@
             "url": "https://developers.facebook.com/plugins",
             "companyId": "meta"
         },
+        "farlight_pte_ltd.": {
```

Comment:
I think we can write it without the point `farlight_pte_ltd.  --> farlight_pte_ltd`